ReactWindows/ReactNative.Shared/Bridge/ReactBridge.cs

```diff
@@ -115,8 +118,17 @@ private void ProcessResponseImmediate(JToken response)
 
         private void ProcessResponse(JToken response, bool isEndOfBatch)
         {
-            // Gets the number of native module calls and validates the batch.
-            var moduleCallCount = GetModuleCallCount(response);
+            int moduleCallCount;
+
+            try
+            {
+                moduleCallCount = GetModuleCallCount(response);
+            }
+            catch (Exception)
+            {
```

Comment:
I don't think we need to catch an exception here. It's not done in the ReactCommon bridge: [https://github.com/facebook/react-native/blob/master/ReactCommon/cxxreact/MethodCall.cpp#L23-L31](https://github.com/facebook/react-native/blob/master/ReactCommon/cxxreact/MethodCall.cpp#L23-L31)

Reply:
I was actually going to follow up there with a PR as well, but I can do it upstream and then bring it back here if that's preferrable.

Reply:
If it's truly a bug... there are pretty strong guarantees about the contract of the `BatchedBridge`, I just assumed that the exceptions thrown might as well have been a Debug assertion.